src/Workspaces/Remote/ServiceHub/Services/SnapshotService.JsonRpcAssetSource.cs

```diff
@@ -50,6 +50,25 @@ public override async Task<IList<(Checksum, object)>> RequestAssetsAsync(int sco
                 }
             }
 
+            public override async Task<bool> IsExperimentEnabledAsync(string experimentName, CancellationToken cancellationToken)
+            {
+                using (RoslynLogger.LogBlock(FunctionId.SnapshotService_IsExperimentEnabledAsync, experimentName, cancellationToken))
```

Comment:
Does this trigger telemetry?  and if so, could this cause too much unnecessary data to be logged?  if some very commonly invoked feature checks if an experiment is enabled each time its invoked, we might get tons of not really useful events.

Reply:
Related to https://github.com/microsoft/vs-streamjsonrpc/issues/277

Reply:
this will fire ETW once Roslyn ETW is enabled but it won't fire VS telemetry.

Reply:
So this is good?

Reply:
yep